index.js

```diff
@@ -7,13 +7,10 @@ module.exports = {
         es6: true,
         commonjs: true,
     },
-    plugins: [
-        "react",
-        "import"
-    ],
+    plugins: ["react", "import"],
     rules: {
         "array-bracket-spacing": [2, "never"],
-        "arrow-body-style": [2, "as-needed", { requireReturnForObjectLiteral: true }],
+        "arrow-body-style": [2, "as-needed", { requireReturnForObjectLiteral: false }],
```

Comment:
I think that's already the default option isn't it? It's fine to leave it explicit like this but you could also remove if you prefer to keep it simple!

Reply:
I don't mind either way.